eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -224,6 +224,7 @@ jobs:
               $account = (Get-AzContext).Account
               $env:AZURESUBSCRIPTION_CLIENT_ID = $account.Id
               $env:AZURESUBSCRIPTION_TENANT_ID = $account.Tenants
+              Update-AzConfig -DisplayBreakingChangeWarning $false
```

Comment:
```suggestion
              Update-AzConfig -DisplayBreakingChangeWarning $false -AppliesTo Az.Accounts
```

Reply:
Also, should this workaround be applied for both FederatedAuth and non-FederatedAuth?